client/src/components/ProfileSettings/ProfileDetails/ProfileDetails.tsx

```diff
@@ -7,84 +19,125 @@ import profCover from '../../../Images/profcover.jpg';
-                                </CardActionArea>
-                                <CardContent className={classes.contentWrapper}>
-                                    <Typography align="center" variant="h4">Norma Byers</Typography>
-                                    <Typography align="center" gutterBottom variant="body1">Loving Pet Sitter</Typography>
-                                    <Typography align="center" gutterBottom variant="body2">
-                                        <LocationOnIcon className={classes.locationIcon} color="secondary" fontSize="small"/>Toronto, Ontario
-                                    </Typography>
-                                    <Typography gutterBottom variant="h4">About me</Typography>
-                                    <Typography align="left" gutterBottom variant="subtitle1" component="h2">
-                                        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Laborum nemo culpa dolore unde illum in quam rerum repellat. Sequi culpa atque,quam laboriosam minus possimus assumenda doloribus distinctio maxime nesciunt!
-                                    </Typography>
-                                    <Box display="flex" p={2} flexDirection="row" flexBasis="1">
-                                        <Avatar className={classes.thumbImages} variant="square" alt="Remy Sharp" src={doggo}/>
-                                        <Avatar className={classes.thumbImages} variant="square" alt="Remy Sharp" src={doggo1}/>
-                                    </Box>
-                                </CardContent>
-                            </Card>
-                        </Paper>
-                    </Grid>
+          <Navbar
+            LOGOUT="/logout"
+            PROFILE="/profile"
+            MY_SITTER="/mySitter"
+            BECOME_SITTER="/becomeSitter"
+            MESSAGE="/message"
+            Logout="Log out"
+            Profile="Profile"
+            Mysitter="My Sitters"
+            Becomesitter="BECOME A SITTER"
+            Messages="Messages"
+          />
+          <Grid item xs={12}>
+            <Grid container direction="row">
+              <Grid item>
+                <Paper elevation={12} className={classes.profileWrapper}>
+                  <Card className={classes.cardWrapper}>
+                    <CardActionArea>
+                      <CardMedia
+                        className={classes.cardImage}
+                        component="img"
+                        alt="profile details img"
+                        height="250"
+                        image={profCover}
+                      />
+                      <Avatar className={classes.cardAvatar} alt="Remy Sharp" src={profimg} />
+                    </CardActionArea>
+                    <CardContent className={classes.contentWrapper}>
+                      <Typography align="center" variant="h4">
+                        Norma Byers
```

Comment:
Since we are integrating the profile details page, can we please remove all the hardcoded data throughout this component?